packages/medusa-payment-stripe/src/services/stripe-provider.js

```diff
@@ -115,14 +123,14 @@ class StripeProviderService extends PaymentService {
   /**
    * Creates a Stripe payment intent.
    * If customer is not registered in Stripe, we do so.
-   * @param {object} cart - cart to create a payment for
-   * @returns {object} Stripe payment intent
+   * @param {Cart} cart - cart to create a payment for
+   * @return {Promise<PaymentSessionData>} Stripe payment intent
    */
   async createPayment(cart) {
```

Comment:
todo: Customer service should use with transaction on line 145
`const customer = await this.customerService_.withTransaction(this.manager_).retrieve(customer_id)`